blocks/cards/cards.js

```diff
@@ -1 +1,90 @@
-/* TODO */
+import {
+  parseFragment,
+  removeEmptyElements,
+  render,
+  convertExcelDate,
+  timeSince,
+  gdPlusIcon,
+  gdPlusIconWhite,
+} from '../../scripts/scripts.js';
+
+let cardData;
+
+async function getCardData() {
```

Comment:
You will probably have to move this into a more generic function in scripts.js when you start working on the carousels.

Have a look at https://github.com/headwirecom/helix-sportsmagazine/blob/main/scripts/lib-franklin.js#L199 for inspiration.